vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+    test: {
```

Comment:
please add basic configuration:
`  test: {
    globals: true,
    environment: "jsdom",
     setupFiles: './vitest.setup.ts',
  } `
  
and then in `./vitest.setup.ts'` add 
import '@testing-library/jest-dom/extend-expect';

Reply:
Done.

According to Vitest page: https://vitest.dev/config/#globals - we should do the following: "To get TypeScript working with the global APIs, add vitest/globals to the types field in your tsconfig.json"..